pkg/cmd/cli/cmd/deploy.go

```diff
@@ -26,6 +26,12 @@ NOTE: This command is still under active development and is subject to change.`
 	deploy_example = `  // Display the latest deployment for the 'database' deployment config
   $ %[1]s deploy database
 
+  // List all deployments for the 'database' deployment config
+  $ %[1]s deploy database -L
```

Comment:
This one will list the history of deployments for the given dc, while describe only gives you the latest one. Also extends the functionality that was already available in the command when running without flags. Sample output:

```
All 6 deployments for test1/database:

  #6 deployed about an hour ago
  #5 deployed about an hour ago
  #4 deployed about an hour ago
  #3 deployed 2 hours ago
  #2 deployed 2 hours ago
  #1 deployed 2 hours ago
```

I do think though we could add more information to the output (or/and add a `--verbose` mode). What would you guys suggest?